examples/mxnet_vision/README.md

```diff
@@ -0,0 +1,184 @@
+
+```bash
+cd mxnet-model-server/examples/mxnet_vision
+curl -O https://s3.amazonaws.com/model-server/model_archive_1.0/examples/squeezenet_v1.1/squeezenet_v1.1-symbol.json
+curl -O https://s3.amazonaws.com/model-server/model_archive_1.0/examples/squeezenet_v1.1/squeezenet_v1.1-0000.params
+```
+
+Alternatively, use these links to download the Symbol and Params files via your browser:
+1. <a href="https://s3.amazonaws.com/model-server/model_archive_1.0/examples/squeezenet_v1.1/squeezenet_v1.1-symbol.json" download>squeezenet_v1.1-symbol.json</a>
+2. <a href="https://s3.amazonaws.com/model-server/model_archive_1.0/examples/squeezenet_v1.1/squeezenet_v1.1-0000.params" download>squeezenet_v1.1-0000.params</a>
+
+## Step 2 - Prepare the signature file
+
+Define Input and Output name, type and shape in `signature.json` file. The signature for this example looks like below:
+
+```json
+{
+  "inputs": [
+    {
+      "data_name": "data",
+      "data_shape": [
+        0,
+        3,
+        224,
+        224
+      ]
+    }
+  ]
+}
+```
+
+In this pre-trained model, input name is 'data' and shape is '(1,3,224,224)'. Where, the expected input is a color image (3 channels - RGB) of shape 224*224. We also expect input type is a binary JPEG images. In provided mxnet_vision_service.py, you will see the code that take care of converting binary images to tensor NDArray used by MXNet.
+
+*Note:* Typically, if you train your own model, you define the Input and Output Layer name and shape when defining the Neural Network. If you are using a pre-trained MXNet model, to get these Input and Output name and dimensions, you can load the Model and extract the Input and Output layer details. Unfortunately, there are no APIs or easy way to extract the Input shape. Example code below:
+
+```python
+>>> import mxnet as mx
+>>> load_symbol, args, auxs = mx.model.load_checkpoint("squeezenet_v1.1", 0)
+>>> mod = mx.mod.Module(load_symbol, label_names=None, context=mx.cpu())
+>>> mod.data_names
+['data']
+>>> mod.bind(data_shapes=[('data', (1, 3, 224, 224))])
+>>> mod.set_params(args, auxs)
+>>> print(mod.data_names)
+>>> print(mod.data_shapes)
+>>> print(mod.output_names)
+>>> print(mod.output_shapes)
+['data']
+[DataDesc[data,(1, 3, 224, 224),<class 'numpy.float32'>,NCHW]]
+['detection_output']
+[('detection_output', (1, 6132, 6))]
+```
+
+## Step 3 - Prepare synset.txt with list of class names
+
+[synset.txt](synset.txt) is where we define list of all classes detected by the model. The list of classes in synset.txt will be loaded by MMS as list of labels in inference logic.
+
+You can use `curl` to download it.
+```bash
+cd mxnet-model-server/examples/mxnet_vision
+
+curl -O https://s3.amazonaws.com/model-server/model_archive_1.0/examples/squeezenet_v1.1/synset.txt
+```
+
+Alternatively, use following link to download:
+<a href="https://s3.amazonaws.com/model-server/model_archive_1.0/examples/squeezenet_v1.1/synset.txt" download>synset.txt</a>
```

Comment:
mark down style hyperlinks than href links?

Reply:
This has to be href. Only this can ensure browser prompt for save to disk.